extension/js/common/core/common.ts

```diff
@@ -353,3 +353,10 @@ export class Url {
 export const emailKeyIndex = (scope: string, key: string): string => {
   return `${scope.replace(/[^A-Za-z0-9]+/g, '').toLowerCase()}_${key}`;
 };
+
+export const asyncSome = async (arr: Array<unknown>, predicate: (e: unknown) => Promise<boolean>) => {
```

Comment:
good. Can we use (Generics)[https://www.typescriptlang.org/docs/handbook/2/generics.html] instead of `unknown`

Reply:
Sounds good. Just changed it